src/clients/ftp-server.rb

```diff
@@ -355,25 +313,25 @@ def main
               "Anonymous users can create directories."
             )
           },
-          #only vsftd
+          # only vsftd
```

Comment:
NP: We were removing allusions to `only vsftpd` below, so why not just remove them completely

Reply:
makes sense